quipucords/api/scanjob/tests_scanjob.py

```diff
@@ -315,6 +315,29 @@ def test_create_invalid_disable_optional_products_val(self):
             'options': {'disable_optional_products':
                         ['Extra vars values must be type boolean.']}})
 
+    def test_create_invalid_start_time(self):
```

Comment:
If users are allowed to pass custom start and end times, is it practical to write a test where they pass correctly-formatted ones?

Reply:
I can write a test to make sure that they can pass correctly formatted ones, but should users ever be able to pass start/end times?

Reply:
(I would think that users cannot set start/end times) which makes my other test cases somewhat useless (haha) because the models will be setting the start/end times. I just wanted to make sure that it failed with invalid values.

Reply:
Should set the serializer as read-only=true.